pkg/snet/arclient/client.go

```diff
@@ -186,66 +165,30 @@ func (c *client) Post(ctx context.Context, path string, payload interface{}) (*h
 		return nil, err
 	}
 
-	addr := c.client.Addr() + path
+	addr := c.httpClient.Addr() + path
 
 	req, err := http.NewRequest(http.MethodPost, addr, body)
 	if err != nil {
 		return nil, err
 	}
 
-	return c.client.Do(req.WithContext(ctx))
-}
-
-// Websocket performs a new websocket request.
-func (c *client) Websocket(ctx context.Context, path string) (*websocket.Conn, error) {
-	header, err := c.client.Header()
-	if err != nil {
-		return nil, err
-	}
-
-	dialOpts := &websocket.DialOptions{
-		HTTPClient: c.client.ReuseClient(),
-		HTTPHeader: header,
-	}
-
-	addr, err := url.Parse(c.client.Addr())
-	if err != nil {
-		return nil, err
-	}
-	switch addr.Scheme {
-	case "http":
-		addr.Scheme = "ws"
-	case "https":
-		addr.Scheme = "wss"
-	}
-
-	addr.Path = path
-
-	conn, resp, err := websocket.Dial(ctx, addr.String(), dialOpts)
-	if err != nil {
-		return nil, err
-	}
-
-	if resp.StatusCode == http.StatusOK {
-		c.client.IncrementNonce()
-	}
-
-	return conn, nil
+	return c.httpClient.Do(req.WithContext(ctx))
 }
 
 // BindRequest stores bind request values.
 type BindRequest struct {
 	Port string `json:"port"`
 }
 
-// BindSTCPR binds client PK to IP:port on address resolver.
-func (c *client) BindSTCPR(ctx context.Context, port string) error {
-	return c.bind(ctx, bindSTCPRPath, port)
+// LocalAddresses contains outbound port and all network addresses of visor.
+type LocalAddresses struct {
+	Port      string   `json:"port"`
+	Addresses []string `json:"addresses"`
 }
 
 // BindSTCPR binds client PK to IP:port on address resolver.
-func (c *client) BindSUDPR(ctx context.Context, port string) error {
-	return c.bind(ctx, bindSUDPRPath, port)
+func (c *client) BindSTCPR(ctx context.Context, tType, port string) error {
```

Comment:
why do we need transport type here? we have this stated in the func name

Reply:
True. It may be removed